clang/include/clang/Sema/Sema.h

```diff
@@ -335,7 +335,8 @@ class SYCLIntegrationHeader {
   ///  Signals that subsequent parameter descriptor additions will go to
   ///  the kernel with given name. Starts new kernel invocation descriptor.
   void startKernel(const FunctionDecl *SyclKernel, QualType KernelNameType,
-                   SourceLocation Loc, bool IsESIMD, bool IsUnnamedKernel);
+                   SourceLocation Loc, bool IsESIMD, bool IsUnnamedKernel,
+                   unsigned long ObjSize);
```

Comment:
`unsigned long` is likely the wrong type for these.  This is 32 bits on windows.  Note that the CharUnits underlying type is `int64_t`.